docs/user/guides/intersphinx.rst

```diff
@@ -1,5 +1,7 @@
-Link to Other Projects' Documentation With Intersphinx
-======================================================
+How to link to other documentation projects with Intersphinx
```

Comment:
We have been using "Title Case" for the main h1 of the document. I'm not sure what's the correct casing, but I wanted to mention that.

Reply:
Ah okay, I wasn't looking a lot around, but the one that I compared it with happened to be not US-Title-Case :)

Reply:
I personally don't really like Title Case Anything because it removes the distinction of proper nouns that are actually supposed to be title case. Plus it's redundant.

Reply:
we have this documented at https://dev.readthedocs.io/en/latest/docs.html#titles.

Reply:
Thanks Santos. Let's follow what we have documented for now. We could open a meta discussion to talk about which approach is better and change the current one if desired.

Reply:
Agreed. Consistency is more important than which option we choose. 👍

Reply:
Agree that consistency is more important.

(Having said that I would generally prefer saving title case only for the highest-level title in an entire documentation set. When written in sentence case:

* titles that appear in lists of contents are easier to scan rapidly and assimilate as a complete phrase 
* they take up marginally less space
* they flow into sentences better when used as inline links
* they allow document titles that are properly capitalised ("Code of Conduct") to stand out as such

I think this also reflects a distinction between US and European traditions in newspaper article titles, so perhaps it's just a matter of cultural preference.)

Reply:
Alright, headline restored to Title Case. Consistency first :100: 

_Wishing the CSS property `text-transform` had `us-title-case` or something like that, would really make it easier if this was all a matter of a CSS transformation._

Reply:
Thanks for the input on title cases @evildmp - I will open an issue internally (this is to ensure we don't manipulate a community member to write a hefty PR change that we can't prioritize before other refactors are done, i.e. Diátaxis)